src/Component/Tests/Symfony/EventListener/FormListenerTest.php

```diff
@@ -75,16 +75,10 @@ public function it_handles_forms(): void
         $request->attributes = $attributes;
         $request->getRequestFormat()->willReturn('html');
 
-        $attributes->get('_route')->willReturn('app_dummy_show');
```

Comment:
I've removed a lot of not nececarry code in this test, but in the end, this form listener and its PHPUnit test class will be removed.